pandas/core/indexes/accessors.py

```diff
@@ -572,6 +572,57 @@ class PeriodProperties(Properties):
 class CombinedDatetimelikeProperties(
     DatetimeProperties, TimedeltaProperties, PeriodProperties
 ):
+    """
+    Combined datetime-like properties for a pandas Series.
+
+    Not meant to be instantiated directly. Instead, used to determine the appropriate
+    parent class (ArrowTemporalProperties, DatetimeProperties,TimedeltaProperties,
+    or PeriodProperties) based on the dtype of the provided pandas Series.
```

Comment:
This class is not made public directly, only via the `Series.dt` accessor. Users don't need to know any of the classes in the description (`ArrowTemporalProperties`...). So, it doesn't make sense to name these classes here as they are internal.

Using the existing `Accessor object for datetimelike properties of the Series values.` description seems to be more appropriate, or you can use `Accessor object for datetime-like, timedelta and period properties of the Series values.`